tests/utils.go

```diff
@@ -1569,12 +1569,16 @@ func RunVMIAndExpectLaunch(vmi *v1.VirtualMachineInstance, timeout int) *v1.Virt
 }
 
 func RunVMIAndExpectLaunchWithDataVolume(vmi *v1.VirtualMachineInstance, dv *cdiv1.DataVolume, timeout int) *v1.VirtualMachineInstance {
-	obj := RunVMI(vmi, timeout)
+	virtClient, err := kubecli.GetKubevirtClient()
+	Expect(err).To(BeNil())
+	_ = RunVMI(vmi, timeout)
 	By("Waiting until the DataVolume is ready")
 	Eventually(ThisDV(dv), timeout).Should(HaveSucceeded())
+	vmi, err = virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
```

Comment:
You could do `vmi, err = ThisVMI(vmi)()` and save some lines.

Reply:
Done, thanks.